dockerfiles/onnxruntime-tensorrt.dockerfile

```diff
@@ -7,39 +7,71 @@
 # Dockerfile to run ONNXRuntime with TensorRT integration
 
 # nVidia TensorRT Base Image
-FROM nvcr.io/nvidia/tensorrt:20.07.1-py3
+FROM nvcr.io/nvidia/tensorrt:20.07.1-py3 AS runtime
 
 ARG ONNXRUNTIME_REPO=https://github.com/Microsoft/onnxruntime
 ARG ONNXRUNTIME_BRANCH=v1.5.3
-
-RUN apt-get update &&\
-    apt-get install -y sudo git bash unattended-upgrades
+ARG DEBIAN_FRONTEND=noninteractive
+ARG PYTHON_VERSION=3.6
 
 WORKDIR /code
 ENV PATH /usr/local/nvidia/bin:/usr/local/cuda/bin:/code/cmake-3.14.3-Linux-x86_64/bin:/opt/miniconda/bin:${PATH}
 ENV LD_LIBRARY_PATH /opt/miniconda/lib:$LD_LIBRARY_PATH
 
+RUN apt update && apt install -y --no-install-recommends \
+        wget \
+        zip \
+        ca-certificates \
+        build-essential \
+        curl \
+        libcurl4-openssl-dev \
+        libssl-dev \
+        python3-dev
+
+RUN curl -fsSL -v -o ~/miniconda.sh -O  https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh  && \
+    chmod +x ~/miniconda.sh && \
+    ~/miniconda.sh -b -p /opt/conda && \
+    rm ~/miniconda.sh && \
+    /opt/conda/bin/conda install -y python=${PYTHON_VERSION} && \
+    /opt/conda/bin/conda install -c pytorch -y cudatoolkit=10.2 && \
+    /opt/conda/bin/conda clean -ya
+RUN python -c "import sys; assert sys.version[:3] == '$PYTHON_VERSION', sys.version"
+RUN pip install numpy
```

Comment:
I think this one would be better to be included in the same command as conda installation together with pytorch. 
and also I think you forgot to include the conda path to env, like this `ENV PATH /opt/conda/bin:$PATH`

Reply:
I dont really agree to include this with conda since the python version assert is after conda, but yes I think that's subjective opinion

Reply:
oh ya need to change miniconda to conda

Reply:
yepp I meant that one